src/vdaf/prio3.rs

```diff
@@ -218,7 +243,7 @@ impl Prio3Aes128Histogram {
 ///     .map(|o| vdaf.aggregate(&(), o).unwrap());
 ///
 /// // Unshard
-/// let agg_res = vdaf.unshard(&(), agg_shares).unwrap();
+/// let agg_res = vdaf.unshard(&(), agg_shares, 5).unwrap();
```

Comment:
nit: to make this more clear, how about changing the for loop above to
```
let measurements = [0, 1, 1, 1, 0];
for measurement in measurements  { ... }
```
... and then ...
```suggestion
/// let agg_res = vdaf.unshard(&(), agg_shares, measurements.len()).unwrap();
```
I think that helps make it clear what this magic value `5` is, especially for people reading this example on docs.rs and not in an editor that can tell them what the third argument to `vdaf.unshard` is.